subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -44,4 +36,10 @@ config NRF_CLOUD_PORT
 config NRF_CLOUD_IPV6
 	bool "Configure nRF Cloud library to use IPv6 addressing. Otherwise IPv4 is used."
 
+module=NRF_CLOUD
+module-dep=LOG
+module-str=Log level for nRF Cloud
+module-help=Enables nRF Cloud log messages.
+source "${ZEPHYR_BASE}/subsys/net/Kconfig.template.log_config.net"
```

Comment:
There's a regular `LOG` template as well, would it be possible to use it instead?
`subsys/logging/Kconfig.template.log_config`

Reply:
Yeah, better do that.